src/components/project/ProjectMonthsLists.tsx

```diff
@@ -118,18 +120,50 @@ type ProjectMonthsListProps = {
 
 /** Display a title and a ProjectMonth list */
 const ProjectMonthsList = ({feature}: ProjectMonthsListProps) => {
+  const projectsMonthOverviews = useSelector((state: ConfacState) => state.projectsMonthOverviews);
+  const dispatch = useDispatch();
+
   if (!feature.list.data.length) {
     return null;
   }
 
-  const model = feature.list.data[0];
-  let month = model.details.month.format('MMMM');
-  month = month.charAt(0).toUpperCase() + month.slice(1);
-  month = t('projectMonth.listTitle', {month, year: model.details.month.year()});
+  const displayMonthWithYear = () => {
+    const {month: date} = feature.list.data[0].details;
+    const formattedMonth = date.format('MMMM').charAt(0).toUpperCase() + date.format('MMMM').substring(1);
+    return t('projectMonth.listTitle', {month: formattedMonth, year: date.year()});
+  };
+
+  const createMonthId = () => {
+    const {month: date} = feature.list.data[0].details;
+    const monthYearFormat = date.format('MMYYYY');
```

Comment:
Ter info: typisch wordt voor zoiets YYYY-MM gebruikt (of YYYYMM)
Dat is sorteerbaar en ook logischer "Largest value - smallest value"

Reply:
Het is misschien logischer dat de projects_month.month property matches de attachments_project_month_overview.monthId?